Scripts/models/generation.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, purpose, resultdata):
 
     def init_tours(self):
         """Initialize `tours` vector to 0."""
-        self.tours = pandas.Series(0, self.purpose.zone_numbers)
+        self.tours = pandas.Series(0.0, self.purpose.zone_numbers)
 
     def add_tours(self):
         """Generate and add (peripheral) tours to zone vector."""
```

Comment:
This may have small impacts on the demand, because I think in the base case the `self.tours` vector will change to `float32` (because that is what is added from `self.zone_data`) in `add_tours`. Now when `self.tours` is implicitly initialized as `float64`, this will probably be broadcasted to large parts of the demand model. I suggest adding a `dtype=numpy.float32` to see if that changes results. This could also be tested in `ExternalModel`.

Reply:
My understanding is that Pandas does not change the Series datatype on addition. In the earlier version the series was created with int64 dtype and each addition of real values would only add the integer part of the number (`floor()`). Comparing to that the difference between float32 and float64 should be minimal. We can add `dtype=numpy.float32` here if want to reduce memory consumption, especially if there are bigger matrices calculated based on this Series and same dtype.

Reply:
We discussed this and it turned out that we were both wrong. `add_tours` has always changed `self.tours` into float64.